packages/relay-compiler/graphql-compiler/package.json

```diff
@@ -7,9 +7,11 @@
   "dependencies": {
     "chalk": "^1.1.1",
     "fb-watchman": "^2.0.0",
-    "graphql": "^0.12.3",
     "immutable": "~3.7.6"
   },
+  "peerDependency": {
```

Comment:
should this be `peerDependencies`?

Reply:
jeez, oops not my finest PR :P

Reply:
Great catch @mattkrick!

@jstejada I don’t know if the bot is still importing, but this new addition will be important.